src/libexpr/eval.cc

```diff
@@ -92,6 +92,16 @@ RootValue allocRootValue(Value * v)
 #endif
 }
 
+// Returns `true' is a string is a reserved keyword which requires quotation
+// when printing attribute set field names.
+static inline bool isReservedKeyword(const std::string_view str)
```

Comment:
```suggestion
static inline std::string_view quoteKeywordAsNeeded(const std::string_view str)
```

Reply:
It might be worth it to arrange to have at compile time the quoted ones ready to go as the above signature implies. Not sure.

Reply:
Could you move this function to `print.cc`?